docs/DeveloperGuide.md

```diff
@@ -650,6 +650,18 @@ The following sequence diagram illustrates how the password management process o
 
 --------------------------------------------------------------------------------------------------------------------
 
+## **Appendix: Known Issues**
+
+### Failing tests on Windows when run more than once
+
+1. In `EncryptionManagerTest.java`, temporary files are created before and deleted after each test.
+2. However, on Windows machine, the test will throw a `java.nio.file.FileSystemException` exception.
```

Comment:
Why?

Reply:
Add brief explanation, maybe link to StackOverflow documentation about this Windows bug.

Reply:
Updated 👍